libweasyl/libweasyl/models/tables.py

```diff
@@ -219,7 +185,7 @@ def default_fkey(*args, **kwargs):
 favorite = Table(
     'favorite', metadata,
     Column('userid', Integer(), primary_key=True, nullable=False),
```

Comment:
If it’s a foreign key, SQLAlchemy won’t make it auto-increment. Explicit does tend to be better than implicit, though – will include in a future cleanup with the rest of them.